### PR TITLE
Deploy to the "opencost" namespace by default

### PR DESCRIPTION
**Pull request by mattray**
Match the behavior from https://www.opencost.io/docs/install

Fixes https://github.com/opencost/opencost-helm-chart/issues/5

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ opencost:
     name: opencost
 
   prometheus:
-    serviceName: my-prometheus
-    namespaceName: opencost
+    serviceName: my-prometheus-server
+    namespaceName: prometheus
```
Comment (Pokom) [inline values.yaml:26]: Suggestion: Can we have these exposed as values so we can override the names?

**Review by Pokom — APPROVED** [review]
Just a minor suggestion on exposing the prometheus config as values. Otherwise :+1:

File: values.yaml (hunk 1 above)
Comment (ts-mini) [reply values.yaml:26]: I did this in my latest PR https://github.com/opencost/opencost-helm-chart/pull/7

Comment (mattray) [reply values.yaml:26]: https://github.com/opencost/opencost-helm-chart/issues/8

**Pull request merged**